smart_home/src/sensors/sensor.cc

```diff
@@ -0,0 +1,9 @@
+#include "../../include/sensors/sensor.h";
+
+namespace sensors {
+
+Sensor::~Sensor() {
+    delete room_;
```

Comment:
same as above, be careful when you delete pointers.. might end up in a double deletion case easily. We'll talk more about it in lesson 5.